apps/system/utils/kdbg_date.c

```diff
@@ -90,9 +94,24 @@ static inline int date_month(FAR const char *abbrev)
 	return ERROR;
 }
 
+#if defined(CONFIG_LIBC_LOCALTIME) || defined(CONFIG_TIME_EXTENDED)
+static inline int day_of_week(FAR const char *abbrev)
+{
+	int wday;
+
+	for (wday = 0; wday < 7; wday++) {
+		if (strncasecmp(g_dayofweek[wday], abbrev, 3) == 0) {
```

Comment:
Will you use string? or number? If you will use string, why do you write like below?
```
printf("                day of week range = [0,6] , (Sunday = 0)\n");
```